connector/openshift/openshift.go

```diff
@@ -61,22 +70,23 @@ type user struct {
 func (c *Config) Open(id string, logger log.Logger) (conn connector.Connector, err error) {
 	ctx, cancel := context.WithCancel(context.Background())
```

Comment:
NB: This line _must_ be removed or changed if/when `Close()` is removed. Otherwise, this leaks resources (`context.WithCancel` spawns a goroutine that only terminates if (a) the parent context expires (will never happen with `context.Background()`), or (b) `cancel()` is called.